test/e2e_node/hugepages_test.go

```diff
@@ -225,12 +225,11 @@ var _ = SIGDescribe("HugePages", framework.WithSerial(), feature.HugePages, "[No
 		restartKubelet(true)
 
 		ginkgo.By("verifying that the hugepages-3Mi resource no longer is present")
-		gomega.Eventually(ctx, func() resource.Quantity {
+		gomega.Eventually(ctx, func() bool {
 			node, err = f.ClientSet.CoreV1().Nodes().Get(ctx, framework.TestContext.NodeName, metav1.GetOptions{})
-			framework.ExpectNoError(err, "while getting node status")
```

Comment:
nit; maybe you want to keep this to avoid hitting null pointers further below?

Otherwise this all lgtm! Thanks!

Reply:
Ah yeah true, this deletion was unintented.